### PR TITLE
[OR-5165] function to generate one-time-link

### PR DESCRIPTION
**Pull request by antoniobarbosa**
- new function to allow api to generate one time link
- Small adjustment in list inquiries to allow params

### CODE REVIEW

File: lib/persona-ruby/actions/inquiries.rb [hunk 1]
```diff
@@ -22,6 +22,10 @@ def inquiry_update(id, attributes = {})
       def inquiry_delete(id)
         connection.delete("inquiries/#{id}", {})
       end
+
+      def inquiry_generate_one_time_link(id, expiration_time = 3600)
```
Comment (jordanell) [inline lib/persona-ruby/actions/inquiries.rb:26]: Maybe just call it "inquiry_one_time_link"?

Comment (jordanell) [inline lib/persona-ruby/actions/inquiries.rb:26]: Maybe do `expiration_time: 3600` instead so we can add hash like options in the future if needed.

**Review by jordanell — CHANGES_REQUESTED** [review]
2 comments just for syntax

File: lib/persona-ruby/actions/inquiries.rb [hunk 2]
```diff
@@ -22,6 +22,10 @@ def inquiry_update(id, attributes = {})
       def inquiry_delete(id)
         connection.delete("inquiries/#{id}", {})
       end
+
+      def inquiry_one_time_link(id, options = {})
```
Comment (antoniobarbosa) [inline lib/persona-ruby/actions/inquiries.rb:26]: I left options for more custom possibilities in the future, and persona already implement a default value in api side

**Review by stredger — APPROVED** [review]

**Review by jordanell — APPROVED** [review]

**Pull request merged**